src/vs/workbench/services/extensions/common/extensions.ts

```diff
@@ -312,6 +312,13 @@ function extensionDescriptionArrayToMap(extensions: IExtensionDescription[]): Ex
 }
 
 export function isProposedApiEnabled(extension: IExtensionDescription, proposal: ApiProposalName): boolean {
+	// --- Start Positron ---
+	// Enable all proposed APIs for selected builtin extensions.
```

Comment:
I think there's a way to see if an extension is built in -- we could use that and just enable API proposals for built-in extensions across the board so we don't need to maintain a list here. Don't really feel strongly about it, though.

Reply:
Good point! Addressed in 29ee97a